package.json

```diff
@@ -1897,14 +1897,21 @@
   },
   "scripts": {
     "vscode:prepublish": "yarn run compile-production",
-    "compile": "yarn install && webpack --mode development --watch --progress",
+    "compile": "yarn install && webpack --mode development",
+    "compile-watch": "yarn install && webpack --mode development --watch --progress",
     "compile-production": "yarn install && yarn run translations-generate && webpack --env BUILD_VSCODE_NLS=true --mode production",
     "translations-export": "gulp translations-export",
     "translations-generate": "gulp translations-generate",
     "translations-import": "gulp translations-import",
     "pretest": "tsc -p test.tsconfig.json",
-    "postinstall": "node ./node_modules/vscode/bin/install",
+    "pr-check": "gulp pr-check",
     "lint": "gulp lint",
+    "unitTests": "tsc -p test.tsconfig.json && node ./out/test/unit-tests/runTest.js",
```

Comment:
instead of invoking `tsc` directly in all these test scripts, can you do `yarn run pretest` instead?